zng/docs/spec.md

```diff
@@ -522,7 +522,7 @@ A TZNG type binding has the following form:
 #<type-tag>:<type-string>
 ```
 Here, `<type-tag>` is a string decimal integer and `<type-string>`
-is a string defining a type according to the [TZNG type grammar`](#42-type-grammar). They create
```

Comment:
Looks like there's a typo here with a stray backtick after "TZNG type grammar"

Reply:
Eek. Thanks for noticing. Fixed.